src/UI/DappModal/components/DappModalHeader.tsx

```diff
@@ -2,17 +2,19 @@ import React from 'react';
 import { faTimes } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import classNames from 'classnames';
-import globalStyles from 'assets/sass/main.scss';
+
 import { WithClassnameType } from '../../types';
+
 import styles from '../dappModalStyles.scss';
+import globalStyles from 'assets/sass/main.scss';
 
-type DappModalHeaderProps = {
+interface DappModalHeaderPropsType extends WithClassnameType {
```

Comment:
export. All interfaces need export because component will show up with props any in consumer

Reply:
Exported. 👍